backend/tsconfig.json

```diff
@@ -26,7 +26,10 @@
 
     /* Modules */
     "module": "commonjs" /* Specify what module code is generated. */,
-    "rootDir": "./src" /* Specify the root folder within your source files. */,
+    "rootDirs": [
+      "./src",
+      "vitest.config.ts"
```

Comment:
the fuck is this

Reply:
มันแจ้งเตือนว่า vitest.config.ts ไม่อยู่ใน rootDir ละมัน build ไม่ผ่าน

Reply:
exclude ก็พอแล้วหนิ